### ISSUE TITLE
t/chown-glob.t broken on macOS

### ISSUE THREAD

**Issue by atoomic** [cpanel/Test-MockFile#98]
This was introduced by d36c34a4badbbf0a6a2877a4e61dbb03c62f72dc

#96 is going to disable this for now, this is a placeholder to fix the test

```
╰─> prove -vl t/chown-glob.t
t/chown-glob.t ..
# Seeded srand with seed '20220120' from local date.
not ok 1 - chown on a file handle works

# Failed test 'chown on a file handle works'
# at t/chown-glob.t line 21.
# +-----+----+-------+
# | GOT | OP | CHECK |
# +-----+----+-------+
# | 0   | eq | 1     |
# +-----+----+-------+
not ok 2 - Owner of the file is now there

# Failed test 'Owner of the file is now there'
# at t/chown-glob.t line 28.
# +-----+----+-------+
# | GOT | OP | CHECK |
# +-----+----+-------+
# | 503 | eq | 10502 |
# +-----+----+-------+
not ok 3 - Group of the file is now there

# Failed test 'Group of the file is now there'
# at t/chown-glob.t line 29.
# +-----+----+-------+
# | GOT | OP | CHECK |
# +-----+----+-------+
# | 0   | eq | 10019 |
# +-----+----+-------+
1..3
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/3 subtests

Test Summary Report
-------------------
t/chown-glob.t (Wstat: 768 Tests: 3 Failed: 3)
  Failed tests:  1-3
  Non-zero exit status: 3
Files=1, Tests=3,  1 wallclock secs ( 0.02 usr  0.00 sys +  0.06 cusr  0.02 csys =  0.10 CPU)
Result: FAIL
```

**Comment by xsawyerx** [comment]
Fixed by #99

**Comment by toddr** [comment]
This test no longer exists.
